app/controllers/presentations_controller.rb

```diff
@@ -37,6 +37,14 @@ def update
     end
   end
 
+  def destroy
+    @presentation = Presentation.find(params[:id])
+    authorize @presentation
+    @presentation.destroy
```

Comment:
Yes. All of the AR Crud methods have a "bang method" equivalent 

so `create` and `create!` 

The normal method returns true or false, the bang method raises an error. Or convention generally is if you are not checking the output of the method, you should raise an error if it fails 

so: 
```ruby
if thing.create
  stuff
else 
  other stuff
end 

# OR 

thing.create! 
```